enclave/Cargo.toml

```diff
@@ -73,19 +73,19 @@ multibase = { git = "https://github.com/whalelephant/rust-multibase", branch = "
 
 # local deps
 itc-light-client = { path = "../core/light-client", default-features = false, features = ["sgx"] }
-pallet-teerex-storage = { path = "../primitives/pallet-teerex-storage", default-features = false }
-substratee-get-storage-verified = { path = "../primitives/get-storage-verified", default-features = false }
-substratee-sgx-crypto = { path = "../primitives/sgx/crypto", default-features = false,  features = ["sgx"] }
-substratee-node-primitives = {path = "../primitives/node", default-features = false, features = ["sgx"] }
-substratee-ocall-api = { path = "../primitives/ocall-api", default-features = false }
-substratee-settings = { path = "../primitives/settings" }
-substratee-sgx-io = { path = "../primitives/sgx/io", default-features = false, features = ["sgx"] }
-substratee-sidechain = { path = "../primitives/sidechain", default-features = false }
-substratee-sidechain-primitives = { path = "../primitives/sidechain/primitives", default-features = false }
-substratee-stf = { path = "../stf", default-features = false, features = ["sgx"] }
-substratee-storage = { path = "../primitives/storage", default-features = false, features = ["sgx"] }
-substratee-worker-primitives= {path = "../primitives/worker", default-features = false }
-test-utils = { path = "../primitives/test-utils", default-features = false, features = ["sgx"] }
+itp-teerex-storage = { path = "../primitives/pallet-teerex-storage", default-features = false }
+itp-storage-verifier = { path = "../primitives/get-storage-verified", default-features = false }
+itp-sgx-crypto= { path = "../primitives/sgx/crypto", default-features = false,  features = ["sgx"] }
+itp-teerex = {path = "../primitives/node", default-features = false, features = ["sgx"] }
```

Comment:
With this, I don't agree. The node-primitives are not only pallet-teerex specific.

Reply:
ok my bad. I had the impression it only contains teerex stuff. suggestion?

Reply:
See comment below, about merging it to itp-primitives.